site/kube-addon.md

```diff
@@ -205,21 +207,16 @@ UDP connection from 10.32.0.7:56648 to 10.32.0.11:80 blocked by Weave NPC.
 
 ### <a name="configuration-options"></a> Changing Configuration Options
 
-The default configuration settings can be changed by saving and editing the
-addon YAML before running `kubectl apply`. Additional arguments may be
-supplied to the Weave router process by adding them to the `command:`
-array in the YAML file.
+#### Using `cloud.weave.works`
 
-Some parameters are changed by environment variables; these can be
-inserted into the YAML file like this:
+If you are getting your YAML file from `cloud.weave.works` as described above, then you can customise it by passing some of Weave Net's options, arguments and environment variables as query parameters:
 
-```
-      containers:
-        - name: weave
-          env:
-            - name: IPALLOC_RANGE
-              value: 10.0.0.0/16
-```
+  - `version`: Weave Net's version. Default: `latest`, i.e. latest release. *WARNING*: Backward compatibility of the YAML is not guaranteed. If you lock Weave Net's version using something else than `latest` for this query parameter, please make sure you do so with the latest release, save a copy of the YAML and use this copy instead of downloading it every time from `cloud.weave.works`.
+  - `known-peers`: comma-separated list of hosts. Default: empty.
```

Comment:
What I meant is:
1. if, for example, the `latest` version is `2.0.0`, but one sets `version=1.9.5`, one will end up with a daemonset configured for `2.0.0` but using Weave Net `1.9.5`'s containers, which is likely to break;
2. therefore one should make sure:
  a. that if `version` is set to something else than `latest` (the default), it is set to the latest version at the time, e.g. in this example, `2.0.0`,
  b. the produced YAML is saved to a local file (e.g. `my.yaml`) and kept (at least until the next upgrade from that user, as for example, tomorrow we might release a change in the LG so that it supports `3.0.0` which in turn, will break `2.0.0`)
  c. install using the file previously saved, e.g. `kubectl -f apply my.yaml`
  d. that way one would have `2.0.0` containers with `2.0.0` YAML / daemonset configuration, and things should be consistent.

This was my attempt to address [this comment](https://github.com/weaveworks/weave/pull/2903#discussion_r117765377).